schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -1394,6 +1395,9 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15-preview/Microsoft.DataMigration.json#/resourceDefinitions/services_projects" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15-privatepreview/Microsoft.DataMigration.json#/resourceDefinitions/services" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-15-privatepreview/Microsoft.DataMigration.json#/resourceDefinitions/services_projects" },
```

Comment:
Shouldn't these references have been added to autogeneratedResources.json automatically?

Reply:
These schemas should have been manually authored. With autogeneration, these resources are not generated. So they were removed in the [onboarding PR](https://github.com/Azure/azure-resource-manager-schemas/pull/1451). Just like how I'm restoring the legacy vaults schema, I also restored these references manually, to prevent similar regression issues.